generators/typescript/sdk/client-class-generator/src/GeneratedSdkClientClassImpl.ts

```diff
@@ -1234,7 +1284,7 @@ export class GeneratedSdkClientClassImpl implements GeneratedSdkClientClass {
         return ts.factory.createPropertyAccessExpression(this.getReferenceToOptions(), option);
     }
 
-    private getOptionKeyForNonLiteralHeader(header: HttpHeader): string {
+    private getOptionKeyForHeader(header: HttpHeader): string {
```

Comment:
Heads up @dsinghvi, noticed that we were using a mixture of `safeName` and `unsafeName`, so I unified everything on `unsafeName` in this helper (for consistency with other parameter types). 

This should be OK, and none of the test fixtures are updated with this change but wanted to flag it in case there's some context I'm missing.